docs/book/src/topics/powervs/creating-a-cluster.md

```diff
@@ -42,6 +42,9 @@ following the steps below.
     > **Note:**
     > The `IBMPOWERVS_IMAGE_ID` value below should reflect the ID of the custom image and the `kubernetes-version` value below should reflect the kubernetes version of the custom image.
 
+    > **Note:**
```

Comment:
instead of a separate note, can we combine the two in the same note with bullet points?